bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/api/AgsMagnitude.scala

```diff
@@ -23,7 +24,7 @@ object AgsMagnitude {
    * Gets the widest possible range limits incorporating the given conditions
    * and speeds.
    */
-  def rangeLimits(mc: MagnitudeCalc, c1: (Conditions, GuideSpeed), c2: (Conditions, GuideSpeed)): MagnitudeRange =
+  private def rangeConstraints(mc: MagnitudeCalc, c1: (Conditions, GuideSpeed), c2: (Conditions, GuideSpeed)): Option[MagnitudeConstraints] =
```

Comment:
It used to be that the operation run on `MagnitudeRange` whose union method returned `MagnitudeRange`, instead it now it returns `Option[MagnitudeConstraint]` as you cannot do a union if e.g. you have different bands. The effects downstream are actually quite limited in `ProbeLimitsCalc` and `GemsMagnitudeTable`